custom_components/tuya_local/devices/leos_loo_too.yaml

```diff
@@ -0,0 +1,86 @@
+name: Leo's Loo Too
+products:
+  - id: 1badzyvwh1e1hrog
+primary_entity:
+  entity: select
+  category: config
+  name: Cleaning Mode
+  dps:
+    - id: 2
+      type: string
+      name: work_mode
+      icon: "mdi:refresh-auto"
+      category: config
+      mapping:
+        - dps_val: auto_clean
+          value: Auto Clean
+        - dps_val: manual_clean
+          value: Manual Clean
+secondary_entities:
+  - entity: switch
+    name: Power
+    category: config
+    icon: "mdi:power"
+    class: switch
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+  - entity: switch
+    name: UV
+    category: config
+    icon: "mdi:alarm-light"
+    class: switch
+    dps:
+      - id: 13
+        name: switch
+        type: boolean
+  - entity: switch
+    name: Light
+    category: config
+    icon: "mdi:lightbulb-outline"
+    class: switch
+    dps:
+      - id: 16
+        name: switch
+        type: boolean
+  - entity: number
+    category: config
+    name: Cleaning Delay
+    dps:
+      - id: 5
+        type: integer
+        name: value
+        range:
+          min: 6
+          max: 1200
+        unit: s
+        step: 3
+  - entity: sensor
+    category: diagnostic
+    name: Last Weight
+    icon: "mdi:scale"
+    class: weight
+    dps:
+      - id: 6
+        type: integer
+        name: cat_weight
+        unit: kg
+        mapping:
+          - scale: 0.1
+  - entity: sensor
+    category: diagnostic
+    name: Time Used (Today)
```

Comment:
Should be "Times used".  I would avoid parentheses in entity names, as the name is also used to generate the entity ids, and special characters may be unpredictably translated with potential for future change.  So maybe today should be part of the name too.